src/pointer.go

```diff
@@ -1,22 +1,27 @@
 package main
 
-import (  
-    "fmt"
+import (
+	"fmt"
 )
 
-func main() {  
-    b := 255
-    var a *int = &b                       //Saving b's address into variable a
-    fmt.Printf("Type of a is %T\n", a)
+func main() {
+	b := 255
+	var a *int = &b //Saving b's address into variable a
+	fmt.Printf("Type of a is %T\n", a)
 	fmt.Println("address of b is", a)
-	
 
-   // Creating pointers using the new function
+	// Creating pointers using the new function
 
-   size := new(int)  // we can imagine that "new" keyword is a pointer
```

Comment:
Please add correct definition of  **new** as its a built in function that allocates memory and return address of it.

Reply:
As already mentioned, the same applies for this one. I totally agree with you but this pull request/issue simply does not address this topic/problem. In order to maintain the issue structure, I would rather deal with this problem in another issue/pull request.

Reply:
>  In order to maintain the issue structure, I would rather deal with this problem in another issue/pull request.

I really appreciate that! Thank you. Merging this request and will open new issues for improvement of code comments.